### DISCUSSION TITLE
I want to patch ox.el for korean translation.

### DISCUSSION THREAD

**Discussion by hwiorn** [kjambunathan/org-mode-ox-odt#120] [Q&A] [answered]
I know you prefer feature request than pull request.
I don't know whether `ox.el` is the part of your enhanced-ox-odt.
But this repository has the `ox.el`.

Should I send this patch mail to org community and make a PR for this again?

**Comment by kjambunathan** [comment]
> I know you prefer feature request than pull request.

That is correct.

>  I don't know whether `ox.el` is the part of your enhanced-ox-odt. But this repository has the `ox.el`.

I won't change anything other than `ox-odt.el` .   I need to have `ox.el` and other `org` repository because I need them for development.


> Should I send this patch mail to org community and make a PR for this again?

Anything that is NOT in ox-odt.el, should be taken up in the core.  




If you want to do that, yes.  When you are doing that you can see the comment here https://github.com/kjambunathan/org-mode-ox-odt/issues/80#issuecomment-939482480, and  the comment right below it where I have recorded my observations (i.e, minor issues) with how org defines it.  You may also want to bring this to the notice of org developers.

That said, There is really no need  to patch up the core at all.  Emacs Org mode crowd is LaTeX crowd, and they will prefer translations that the LaTeX engine prefers, and it may happen that what LibreOffice does by default or the translations that you actually want is different from what the LaTeX engine does.  To give you an example, images in LibreOffice  are called Illustrations, but Org prefers to call the images as figures.  Since we are working with Emacs,   one can setup the things the way one wants in one's own .emacs.  See `org-export-dictionary--add-language` in https://emacs.stackexchange.com/a/64150.  

Whatever you do please share your translations here.  I guess LibreOffice is more popular in East Asia (Japan, Korea, China, Taiwan) or South East Asia (Vietnam, Thai) and other Asian and Middle-East countries.  
 
I wouldn't mind if you post a link to this repo, with some words about the enhancements you find useful like Uwe Brauer did here https://lists.gnu.org/archive/html/emacs-orgmode/2021-07/msg00376.html.  Some people in the emacs-orgmode list aren't going to like it, or even resent it and be vocal about their hatred.  If you don't mind a little bit of backlash, by posting a link to this repo or to the ELPA repo of this repository,  you will be **informing people who rely on ODT exporter what the best place for getting help for LibreOffice exporter is**.

**Reply by hwiorn** [reply]
> If you want to do that, yes. When you are doing that you can see the comment here [#80 (comment)](https://github.com/kjambunathan/org-mode-ox-odt/issues/80#issuecomment-939482480), and the comment right below it where I have recorded my observations (i.e, minor issues) with how org defines it. You may also want to bring this to the notice of org developers.
> 
> That said, There is really no need to patch up the core at all. Emacs Org mode crowd is LaTeX crowd, and they will prefer translations that the LaTeX engine prefers, and it may happen that what LibreOffice does by default or the translations that you actually want is different from what the LaTeX engine does. To give you an example, images in LibreOffice are called Illustrations, but Org prefers to call the images as figures. Since we are working with Emacs, one can setup the things the way one wants in one's own .emacs. See `org-export-dictionary--add-language` in https://emacs.stackexchange.com/a/64150.

Thank you for your thoughtful information.

> Whatever you do please share your translations here.

I will. :)

> I guess LibreOffice is more popular in East Asia (Japan, Korea, China, Taiwan) or South East Asia (Vietnam, Thai) and other Asian and Middle-East countries.

Actually, LO is not so popular in Korea. Most companies just use MS office and Windows. I mean MS Office is already standard de factor and there are some compatible office softwares in Korea. LO is an alternative open source solution. I know it's the same in Japan though. My purposes are to use Linux and Emacs at my work and to generate compatible document which can co-work with my collegues. 

If the document was my own publishing, I would just export as LaTeX without any doubt. I tried LaTeX as company doc-form before. But, I realized that making LaTeX templates is a pain and so much time consuming when editing delicate layout design to target, then I gave up.

**Comment by kjambunathan** [comment, marked as answer]
> Should I send this patch mail to org community and make a PR for this again?

The answer is take it up with Org mode maintainers.   _(I don't know if my long-winded answer confused you)_
